### PR TITLE
Update the popover=hint explainer to *not* talk about invoker/interest

### PR DESCRIPTION
**Pull request by mfreed7**
This PR removes the sections that proposed ways to hover-trigger popovers and handle de-hover-hiding of popovers. That is covered (now) by the https://open-ui.org/components/invokers.explainer explainer.

Along the way, I added a lot of additional information about tooltips, and went into more detail about the behaviors.

### CODE REVIEW

**Comment by mfreed7** [comment]
@scottaohara I promised I'd ping you when I had this PR up. I then forgot about that in the intervening weeks. Here's your belated ping. 😄

**Comment by mfreed7** [comment]
Also @dbaron or @gregwhitworth a review would be great when you have a chance.

File: site/src/pages/components/popover-hint.research.explainer.mdx [hunk 1]
```diff
@@ -37,120 +34,94 @@ A followup proposal to the [original Popover proposal](https://open-ui.org/compo
...
+Definition <span>#</span>2 is mis-guided. Many things can be "hover-triggered" without being tooltips. A great example is a hover-triggered menu, where hovering over a sub-menu item brings up the sub-menu. Most people agree that the sub-menu is **not** a tooltip. It just happens to be hover-triggered.
 
-## Add the `interest` value
+That leaves definition <span>#</span>3, which I believe is the correct definition. A tooltip should not contain directly-necessary information - that should be placed into the page content directly. It should provide auxiliary or extra information that is available only when the user wants to dig deeper into some content. And that extra information should be able to contain more than just text, since the auxiliary information might very well contain or even **be** an image or other non-text content.
```
Comment (gregwhitworth) [inline site/src/pages/components/popover-hint.research.explainer.mdx:55]: I'd recommend removing `I` from this statement. Possibly something like, "which is the recommended definition". You should probably open an issue to get alignment on the definition and upon resolution make this *the* definition and move the other considerations to an appendix.

File: site/src/pages/components/popover-hint.research.explainer.mdx [hunk 2]
```diff
@@ -37,120 +34,94 @@ A followup proposal to the [original Popover proposal](https://open-ui.org/compo
...
 
-To support hover triggering, a new value is added: `popovertargetaction=interest`. The name of "interest" was [chosen](https://github.com/openui/open-ui/issues/767#issuecomment-1654177227) to represent the fact that the popover will be triggered not just via mouse-hover, but also by keyboard focus or touchscreen long-press. When set to the `interest` value, the popover will be **shown** when the invoking element is **a)** hovered with the mouse, **b)** focused, or **c)** long-pressed (mobile).
+Note, importantly, that a tooltip **does not need** to be triggered by hover to be a tooltip. Also note that all things that are hover-triggered are not tooltips. A tooltip is defined more by **its content** than how it gets **activated**.
```
Comment (gregwhitworth) [inline site/src/pages/components/popover-hint.research.explainer.mdx:57]: remove `more` as it's not adding anything

File: site/src/pages/components/popover-hint.research.explainer.mdx [hunk 3]
```diff
@@ -16,18 +16,15 @@ A followup proposal to the [original Popover proposal](https://open-ui.org/compo
...
-- ["Hover" triggering](#hover-triggering)
-  - [Add the `interest` value](#add-the-interest-value)
-  - [CSS values to control delay](#css-values-to-control-delay)
-  - [Example buttons](#example-buttons)
-  - [Popover types](#popover-types)
-  - [Example menu](#example-menu)
-- [`popover=hint` Behavior](#popoverhint-behavior)
+  - [What is a tooltip/toggletip](#what-is-a-tooltiptoggletip)
+  - [Tooltip lifecycle behaviors](#tooltip-lifecycle-behaviors)
+- [This Explainer is only about #2](#this-explainer-is-only-about-2)
```
Comment (gregwhitworth) [inline site/src/pages/components/popover-hint.research.explainer.mdx:21]: This title seems odd. Can you adjust it to say something like "Scope of this explainer" and then note that 2 & 3 are out of scope for this document.

**Review by gregwhitworth — APPROVED** [review]
Left some general feedback

File: site/src/pages/components/popover-hint.research.explainer.mdx (hunk 1 above)
Comment (mfreed7) [reply site/src/pages/components/popover-hint.research.explainer.mdx]: Yeah good point. I changed it to roughly your suggestion, plus some links.

File: site/src/pages/components/popover-hint.research.explainer.mdx (hunk 2 above)
Comment (mfreed7) [reply site/src/pages/components/popover-hint.research.explainer.mdx]: Removed `more` but had to re-jigger the ending to be parallel.

File: site/src/pages/components/popover-hint.research.explainer.mdx (hunk 3 above)
Comment (mfreed7) [reply site/src/pages/components/popover-hint.research.explainer.mdx]: Done.

**Review by mfreed7 — COMMENTED** [review]
Thanks for the suggestions! I believe I addressed them all.

**Comment by mfreed7** [comment]
Ok, I'm going to land this, but if there are any issues, I'm happy to fix them.

**Pull request merged**